iree/turbine/kernel/wave/templates/conv.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import iree.turbine.kernel as tk
+import iree.turbine.kernel.lang as tkl
+import iree.turbine.kernel.wave as tkw
+from typing import Any
+from iree.turbine.kernel.lang.global_symbols import *
+
+
+def get_igemm_conv2d(
+    layout: str,
+    n: int,
+    h: int,
+    w: int,
+    c: int,
+    hf: int,
+    wf: int,
+    nf: int,
+    stride: int,

```

Comment:
can we add block_m, block_n, block_k, ratio_m, ratio_n as optional arg with default values?

Reply:
We can add assertions and heuristics in upcoming PR :)

Reply:
Done